.circleci/config.yml

```diff
@@ -705,4 +707,4 @@ jobs:
       - upload_binaries_command:
           platform: << parameters.platform >>
       - slack/notify:
-          <<: *slack-fail-event
+          <<: *slack-fail-event
```

Comment:
Missing a newline at end of the file.

Reply:
Oh interesting. It's probably because of my "whitespace" settings on my editor. Is it better to add a newline back into the end of the file?

Reply:
I updated my VSCode settings. Thanks for the catch.